doc/manual/src/json/derivation.md

```diff
@@ -0,0 +1,65 @@
+{{#include ./experimental-notice.md}}
+
+The JSON serialization of a
+[store derivation](@docroot@/glossary.md#gloss-store-derivation)
```

Comment:
```suggestion
The JSON serialization of a set of
[store derivations](@docroot@/glossary.md#gloss-store-derivation)
```

Otherwise the pluralisation in the rest of the sentence doesn't fit.